VKClone/ViewControllers/MainViewController.swift

```diff
@@ -45,16 +50,68 @@ final class MainViewController: UITableViewController {
     // MARK: - Private methods
 
     private func fetchMyFriend() {
-        clearTableViewAction()
-        apiService.fetchMyFriends { [weak self] friends in
+        apiService.fetchMyFriends { [weak self] isSaved in
             guard let self = self else { return }
-            self.myFriendsFromApi = friends
-            self.sortByLettersVK(array: friends)
-            self.tableView.reloadData()
+            if isSaved {
+                self.myFriendsFromApi = self.loadMyFriendsFromDB()
+                self.sortByLettersVK(array: self.myFriendsFromApi)
+                self.registerDBUpdateToken()
+            }
+        }
+    }
+
+    private func loadMyFriendsFromDB() -> [VKFriend] {
+        do {
+            let realm = try Realm()
+            let friends = realm.objects(VKFriend.self)
+            return Array(friends)
+        } catch {
+            print(error)
+            return []
+        }
+    }
+
+    private func registerDBUpdateToken() {
+        do {
+            let realm = try Realm()
+            let friends = realm.objects(VKFriend.self)
+            myFriendsUpdatedToken = friends.observe { [weak self] changes in
+                guard let self = self else { return }
+                for frnd in friends {
+                    print("friend \(frnd.firstName) online value is \(frnd.online)")
```

Comment:
```suggestion
                    print("\(frnd.firstName)  \(frnd.online)")
```
текстовки в константы